Userland/Libraries/LibC/arpa/nameser.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <inttypes.h>
+#include <netinet/in.h>
+#include <sys/cdefs.h>
+#include <sys/socket.h>
+
+__BEGIN_DECLS
+
+// TODO(cwasser): Stubs to compile the Qt Network module
```

Comment:
We don't use this TODO style per the coding style document. A simple `// FIXME:` will do. In this case, probably a FIXME that points to wherever the rest of the declarations in this file are supposed to be sourced from.

Is there a spec for these declarations? I couldn't find one in the posix spec pages. Some of these names are annoyingly generic. Especially `HEADER` :|

Reply:
These are needed here: https://github.com/qt/qtbase/blob/8222e06d12a4e2a84ce1161abd0ceb58622c740b/src/network/kernel/qdnslookup_unix.cpp#L195
I just discovered that I can disable the `dnslookup` feature so that this file doesn't even get compiled.